index.html

```diff
@@ -3,10 +3,139 @@
+            </div>
+            <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank" rel="noopener"><p class="seefullprogram">See Full Program</p></a>
+        </section>
+        <section class="featuredcomedians dflex flexcolumn py64">
+            <p class="alignselfcenter">Featured Comedians</p>
+            <span class="spanline1 alignselfcenter"></span>
+            <div class="comediancontainer dflex flexcolumn">            
+                <!-- <div class="Comedian dflex">
+                    <a href="https://www.instagram.com/mattrife/" target="_blank"><img class="instagramicon" src="img/mattrife.png" alt="Instagram"></a>
+                    <div class="comediantext dflex flexcolumn ">
+                        <h2 class="comedianname">Matt Rife</h2>
+                        <h2 class="mp0">Comedian</h2>
+                        <p class="mp0">Standup Comedian/Actor/Person. <br> Stop taking the internet so seriously❤️ <br> TOUR DATES 👇🏼 <br> mattrifeofficial.com</p>
+                    </div>
+                </div>
+                <div class="Comedian dflex">
+                    <a href="https://www.instagram.com/jarcuri/" target="_blank"><img class="instagramicon" src="img/jeffacuri.png" alt="Instagram"></a>
+                    <div class="comediantext dflex flexcolumn">
+                        <h2 class="comedianname">Jeff Arcuri</h2>
+                        <h2 class="mp0">Comedian</h2>
+                        <p class="mp0">New Clips M/W/F <br> Booking: KValentine <br> @wmeagency.com <br> MLevy@wmeagency.com <br> Email list/Ticket links👇 <br> linktr.ee/jeffarcuri</p>
+                    </div>
+                </div> -->
+            </div>
+            <div class="comediancontainer dflex flexcolumn">
+                <!-- <div class="Comedian dflex">
+                    <a href="https://www.instagram.com/lukekidgell/" target="_blank"><img class="instagramicon" src="img/lukekidgell.png" alt="Instagram"></a>
+                    <div class="comediantext dflex flexcolumn">
+                        <h2 class="comedianname">Luke Kidgell</h2>
+                        <h2 class="mp0">Comedian</h2>
+                        <p class="mp0">Half of @luke.and.meg <br> Touring Aus 2023 🇦🇺 <br> ⬇️ <br> linktr.ee/lukekidgell</p>
+                    </div>
+                </div>
+                <div class="Comedian dflex">
+                    <a href="https://www.instagram.com/reubensolo/" target="_blank"><img class="instagramicon" src="img/reubensolo.png" alt="Instagram"></a>
+                    <div class="comediantext dflex flexcolumn">
+                        <h2 class="comedianname">Reuben Solo</h2>
+                        <h2 class="mp0">Comedian</h2>
+                        <p class="mp0">🇦🇺 Aus and 🇬🇧 UK tour on sale <br> Next up: Melbourne <br> TICKETS ⤵️ <br> www.reubensolo.com</p>
+                    </div>
+                </div> -->
+            </div>
+
+            
```

Comment:
- [ ] And kindly implement this and/or give the mobile version an appropriate `footer` 🙂


<img width="128" alt="image" src="https://user-images.githubusercontent.com/41163571/233210303-3efc3734-9efe-4199-a9e7-80bf64b55b03.png">